NEWS.md

```diff
@@ -8,6 +8,8 @@
 
 * The family of `update*Input()` functions can now render HTML content passed to the `label` argument (e.g., `updateInputText(label = tags$b("New label"))`). (#3996)
 
+* The `callback` argument of Shiny.js' `InputBinding.subscribe()` method gains support for a value of `"event"`. This makes it possible for an input binding to use event priority when updating the value (i.e., send immediately and always resend, even if the value hasn't changed). (#4211)
```

Comment:
It's probably too much for the NEWS bullet, but the PR itself should include at least one example using the new `priority: "event"` option.